yt_dlp/extractor/rai.py

```diff
@@ -51,6 +51,10 @@ def _extract_relinker_info(self, relinker_url, video_id, audio_only=False):
                 query={'output': 45, 'pl': platform},
                 headers=self.geo_verification_headers())
 
+            if xpath_text(relinker, './license_url', default='{}') != '{}':
+                self.report_drm(video_id)
+                continue
```

Comment:
Error should be raised only if ALL formats have DRM, correct?

Reply:
DRM is detected and, like geoprotection, is assumed is the same for all formats.

'continue' is a leftover from testing, I'll remove it.